.ci/install_qemu.sh

```diff
@@ -8,23 +8,24 @@
 set -e
 
 QEMU_REPO="github.com/qemu/qemu"
-QEMU_DIR="qemu"
 KATA_QEMU_BRANCH="stable-2.11"
 PACKAGING_REPO="github.com/kata-containers/packaging"
-QEMU_CONFIG_SCRIPT="configure-hypervisor.sh"
+QEMU_CONFIG_SCRIPT="${GOPATH}/src/${PACKAGING_REPO}/scripts/configure-hypervisor.sh"
```

Comment:
Remember always that GOPATH is potentially a colon separated list of directories, or may not even exist (in which case the go tooling uses $HOME/go).

I suggest

      MYGOREPO=${GOPATH:-"$HOME/go"}
      MYGOREPO=${MYGOREPO%%:*}
      QEMU_CONFIG_SCRIPT="${MYGOREPO}/src/....

Reply:
That's an interesting point now they have changed the go path rules indeed. I think we are currently 'safe' here as this only runs (at present) in our CI, which always sets the GOPATH to a single path - but, good points.

Reply:
@icarus-sparry  thx, I removed the eval call, for GOPATH we are still safe, but we may want to rework all our ci scripts that relies in this variable